train_diffusion_prior.py

```diff
@@ -41,16 +41,64 @@ def eval_model(model,device,image_reader,text_reader,start,end,batch_size,loss_t
         avg_loss = (total_loss / total_samples)
         wandb.log({f'{phase} {loss_type}': avg_loss})
 
-def save_model(save_path, state_dict):
+def save_diffusion_model(save_path, state_dict):
     # Saving State Dict
     print("====================================== Saving checkpoint ======================================")
     torch.save(state_dict, save_path+'/'+str(time.time())+'_saved_model.pth')
+    
+def load_diffusion_model(dprior_path, 
```

Comment:
let's put that in dalle2_pytorch.py so it can be reused

Reply:
Fixed.